src/searching.md

```diff
@@ -3,17 +3,16 @@
 <!-- toc -->
 
 Anki's Browse screen and the Filtered Deck feature use a common method
-of searching for specific cards/notes.
+of searching for specific cards/notes. This method is also used in the search bar below "FSRS parameters" field.
 
 ## Simple searches
 
 When you type some text into the search box, Anki finds matching notes
 and displays their cards. Anki searches in all fields of the notes, but
-does not search for tags (see later in this section for how to search
-for tags). Some examples:
+does not search for tags (For information on how to search for tags, see [this section](##tags-decks-cards-and-notes) of the page). Some examples:
 
 `dog`\
-search for "dog" - will match words like "doggy" and "underdog" too.
+searches for "dog" - will match words like "doggy" and "underdog" too.
```

Comment:
The conjugation will feel consistent otherwise. In the very next phrase we say "will match words" as if to say "the search term will match words like...". It is only appropriate to use searches instead of search, to mean "this term searches for 'dog'. Using  "search" will make it sound like an imperative sentence, as in, "Do this" or "Search that", or a TV ad "use our new...".

Reply:
I see how that's another fair reading of it.